src/engine/fileOpsIns.cpp

```diff
@@ -104,7 +160,7 @@ void DivEngine::loadDMP(SafeReader& reader, std::vector<DivInstrument*>& ret, St
       }
     } catch (EndOfFileException& e) {
       lastError="premature end of file";
-      logE("premature end of file!");
+      logE(lastError.c_str());
```

Comment:
Nope - not a good idea, especially if there will be format specifiers on the error string.
I will simplify this approach later...

Reply:
Reverted.